### DISCUSSION TITLE
[Bug]: kvsWebrtcClientMasterGstSample receive audio not work

### DISCUSSION THREAD

**Discussion by overheat** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1990] [Q&A]
### Please confirm you have already done the following

- [X] I have searched the repository for related/existing bug reports
- [X] I have all the details the issue requires

### Please answer the following prompt

- [X] This issue is replicable using the unmodified sample application

### Describe the bug

master(camera) not play received audio.

### Expected Behavior

master(camera) play received audio.

### Current Behavior

master(camera) not play received audio.

### Reproduction Steps

Master:`./samples/kvsWebrtcClientMasterGstSample test audio-video devicesrc`
Viewer: test page

### WebRTC C SDK version being used

1.10.2 (#1981)

### Compiler and Version used

gcc 13.2

### Operating System and version

master:ubuntu 24.04

### Platform being used

master:PC x64

**Comment by disa6302** [comment]
@overheat ,

You can receive media by setting `pSampleConfiguration->receiveAudioVideoSource` field to your receive media implementation . Is this something you are already doing?

**Reply by overheat** [reply]
I tried this, but not works for kvsWebrtcClientMasterGstSample on my Ubuntu 24.04 laptop.
BTW, webrtc test page works for Master received audio.
